### ISSUE TITLE
[BUG] Tsfe.php init ConfigurationPageResolver.php that required TwoLevelCache.php without the necessary parameter ($cacheName)

### ISSUE THREAD

**Issue by brannow** [TYPO3-Solr/ext-solr#3505]
**Describe the bug**

0. Typo-3 Backend
1. Open a certain kind of Custom Extension Record that is indexed by SOLR
2. Click Save in the Top of the Typo3 Backend
3. See error

```
Too few arguments to function ApacheSolrForTypo3\Solr\System\Cache\TwoLevelCache::__construct(), 0 passed in /var/www/html/app/web/typo3/sysext/core/Classes/Utility/GeneralUtility.php on line 3249 and at least 1 expected

in /var/www/html/app/web/typo3conf/ext/solr/Classes/System/Cache/TwoLevelCache.php line 53
```

Analysed the Problem a Little bit.
The Main Problem is that in [Tsfe.php](https://github.com/TYPO3-Solr/ext-solr/blob/11.5.1/Classes/FrontendEnvironment/Tsfe.php#L371) is [ConfigurationPageResolver.php](https://github.com/TYPO3-Solr/ext-solr/blob/11.5.1/Classes/System/Configuration/ConfigurationPageResolver.php#L48) created and [ConfigurationPageResolver.php](https://github.com/TYPO3-Solr/ext-solr/blob/11.5.1/Classes/System/Configuration/ConfigurationPageResolver.php#L48) create [TwoLevelCache.php](https://github.com/TYPO3-Solr/ext-solr/blob/11.5.1/Classes/System/Cache/TwoLevelCache.php#L53) via Constructor Argument Injection.

I looked up the Services.yaml but did not found any Default values Defined for [TwoLevelCache.php](https://github.com/TYPO3-Solr/ext-solr/blob/11.5.1/Classes/System/Cache/TwoLevelCache.php#L53)::$cacheName

in short: Tsfe.php called via makeInstance, ConfigurationPageResolver.php. ConfigurationPageResolver.php called via Constructor DI, TwoLevelCache.php. TwoLevelCache.php Requires a Scalar Argument ($cacheName). Which is in this Szenario not Provided.

 - TYPO3 Version: 11.5.24
 - Browser: ANY
 - EXT:solr Version: 11.5.1
 - Used Apache Solr Version: ANY
 - PHP Version: 8.1
 - MySQL Version: ANY

**Comment by brannow** [comment]
The easiest solution would be 

[Tsfe.php#L371](https://github.com/TYPO3-Solr/ext-solr/blob/11.5.1/Classes/FrontendEnvironment/Tsfe.php#L371)

```php
$configurationPageResolver = GeneralUtility::makeInstance(ConfigurationPageResolver::class);
```
changed to 
```php
$configurationPageResolver = GeneralUtility::makeInstance(ConfigurationPageResolver::class, null, null);
```
